free-1.md

```diff
@@ -1,68 +1,98 @@
+Something that would be difficult is to track their meals. Since there are a lot of apps that allow the user to record their meals, I feel that there's no need for this DSL to do so. This DSL's featuers will mostly focus on helping the user do something about their diet/meals beforehand, rather than afterwards. 
+
 
 ### Related work
 _Are there any other DSLs in this domain? If not, describe how you know there
-aren't and conjecture why not. If so, describe them and provide links. How well 
-do they address the need? Are there any particularly admirable qualities of the
-language? Are there parts of the language you think could be improved?_
+aren't and conjecture why not. If so, describe them and provide links. How well do they address the need? Are there any particularly admirable qualities of thelanguage? Are there parts of the language you think could be improved?_
 
+One of the apps I mentioned I tried out is "myfitnesspal" (https://www.myfitnesspal.com/), which is essnetially a calorie counter. It makes it easy for the user to track what they have been eating and count calories they've been consuming. However, again, it does not let the user plan meals beforehand or evaluate how they are doing on their diet based on anything other than number of calories.
 
 ## The Project
 This section examines whether the idea makes for a good CS 111 project.
 
 
 ### Suitability
-_If someone were to work on this project, what percentage of their time would be
-spent directly engaging in the **language** aspects of this project (e.g.,
+_If someone were to work on this project, what percentage of their time would be spent directly engaging in the **language** aspects of this project (e.g.,
 making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I believe building the systems aspects of the project would take slightly longer than the language aspects. 
+A public API for myfitnesstracker or other calorie counters might be used.
+
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+The project is not that ambitious in a sense that it does not have too many features or complicated settings, since the goal of the project is to let the user specify their needs intuitively and very simply. However, I can imagine this DSL being integrated into some of the calories counter/weight loss apps.
 
 ### Benefits and drawbacks
-_Why might this be a good idea for a project? Why might this not be a good idea 
-project?_
+_Why might this be a good idea for a project? Why might this not be a good idea project?_
+
+I believe this project is suitable for the DSL class, because its domain is very narrow and specific. 
+I can imagine running into problems when trying to use the API of the calorie counter/weight loss apps due to their limited access for the public.
```

Comment:
See above comments regarding my hesitation to create a programmatic diet description and its health hazards.